src/main/resources/jupyter/rstudio-docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+services:
+  rstudio:
+    container_name: "${RSTUDIO_SERVER_NAME}"
+    image: "${RSTUDIO_DOCKER_IMAGE}"
+    network_mode: host
```

Comment:
I'm wondering if we should stop using `network_mode: host` at some point and start explicitly managing ports. We've been lucky there are no conflicts so far. Probably out of scope for this PR.

Reply:
In fact there's even a ticket: https://github.com/DataBiosphere/leonardo/issues/154

Reply:
i'll leave this for that ticket but agree that we should start doing this soon

Reply:
sounds good